src/scales/colorScale.ts

```diff
@@ -6,6 +6,7 @@ export module Scale {
 
     private static HEX_SCALE_FACTOR = 20;
     private static LOOP_LIGHTEN_FACTOR = 1.6;
+    private static MAXIMUM_COLORS = 256;
```

Comment:
Probably should give this a name that indicates it's the maximum number of colors when loading them via CSS; the user can still set more colors using `.range()`.